Update conditional-access-integrate-jamf.md (#2626)
MicrosoftDocs/memdocs: memdocs/intune/protect/conditional-access-integrate-jamf.md

- Jamf Pro 10.1.0 or later
- Microsoft Intune and Microsoft AAD Premium P1 licenses (recommended Microsoft Enterprise Mobility + Security license bundle)

> - Global Administrator role in Azure Active Directory.

Reviewer: ```suggestion
- Global admin role in Azure Active Directory.
```

8. Next, we will add permissions to update device attributes. At the top left of the **API permissions** page, select **Add a permission** to add a new permission. 

9. On the **Request API permissions** page, select **Intune**, and then select **Application permissions**. Select only the check box for **update_device_attributes** and save the new permission.

> 10. Under **Microsoft Graph**, click **Application permissions**, and then select **Application.Read.All**.

Reviewer: ```suggestion
10. Under **Microsoft Graph**, select **Application permissions**, then select **Application.Read.All**.
```

> 11. Click **Add permissions**.

Reviewer: ```suggestion
11. Select **Add permissions**.
```

> 12. Navigate to APIs my organization uses. Search for and click **Windows Azure Active Directory**. Click **Application permissions**, and then select **Application.Read.All**.

Reviewer: ```suggestion
12. Navigate to **APIs my organization uses**. Search for and select **Windows Azure Active Directory**. Select **Application permissions**, and then select **Application.Read.All**.
```

> 13. Click **Add permissions**.

Reviewer: ```suggestion
13. Select **Add permissions**.
```

14. Next, grant admin consent for this app by selecting **Grant admin consent for _\<your tenant>_** in the top left of the **API permissions** page. You may need to re-authenticate your account in the new window and grant the application access by following the prompts.  

> 15. Refresh the page by click on the **Refresh** button at the top of the page. Confirm that admin consent has been granted for the **update_device_attributes** permission. 

Reviewer: ```suggestion
15. Refresh the page by selecting **Refresh** at the top of the page. Confirm that admin consent has been granted for the **update_device_attributes** permission. 
```

1. Activate the connection in the Jamf Pro console:

>    1. Open the Jamf Pro console and navigate to **Global Management** > **Conditional Access**. Click the **Edit** button on the **macOS Intune Integration** tab.

Reviewer: ```suggestion
   1. Open the Jamf Pro console and navigate to **Global Management** > **Conditional Access**. Select **Edit** on the **macOS Intune Integration** tab.
```

   2. Select the check box for **Enable Intune Integration for macOS**. When this setting is enabled, Jamf Pro sends inventory updates to Microsoft Intune. Clear the selection if you want to disable the connection but save your configuration.

>    3. Select **Manua** under Connection Type.

Reviewer: ```suggestion
   3. Select **Manual** under **Connection type**.
```

   4. From the **Sovereign Cloud** pop-up menu, select the location of your Sovereign Cloud from Microsoft.

>    5. Click **Open administrator consent URL**, and follow the onscreen instructions to allow the Jamf Native macOS Connector app to be added to your Azure AD tenant.

Reviewer: ```suggestion
   5. Select **Open administrator consent URL** and follow the onscreen instructions to allow the Jamf Native macOS Connector app to be added to your Azure AD tenant.
```
